### ISSUE TITLE
Command executed with exception: Published

### ISSUE THREAD

**Issue by tzachshabtay** [tschaub/gh-pages#221]
Hi, using the command line utility from AppVeyor CI, the publish succeeds but I'm getting an error:

```
gh-pages : Published
At line:10 char:5
+     gh-pages -d Docs\_site -m "CI Docs Updates"
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Published:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Command executed with exception: Published
```

Looking at the code, I'm guessing the problem is that you write the "Published" message to stderr and not stdout: 

https://github.com/tschaub/gh-pages/blob/ec1736a6a2cc6f9a3eec738ac05146f0e3f3e232/bin/gh-pages#L65

Full Appveyor log: https://ci.appveyor.com/project/tzachshabtay/monoags/build/1.0.383/job/gxm2yqmkeg996nu1

Appveyor script that triggers the error:
https://github.com/tzachshabtay/MonoAGS/blob/master/appveyor.yml

**Comment by tschaub** [comment]
Hi @tzachshabtay - what is "the command line utility from AppVeyor CI"?

**Comment by tzachshabtay** [comment]
@tschaub you can see from the script I posted above, I just call:

```
gh-pages -d Docs\_site -m "CI Docs Updates"
```

**Comment by tschaub** [comment]
Should have been fixed by #264.
